### PR TITLE
Update sequel's destroy event

### PR DESCRIPTION
**Pull request by EugeneYak**

### CODE REVIEW

**Comment by 0exp** [comment]
Need to update `CHANGELOG.md` too

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [2.0.0] - 2019-04-01
+### Changed
+- Published: When using Sequel all events checks `if` and `unless` predicates.
```
Comment (0exp) [inline CHANGELOG.md:6]: *Sequel publishing hooks*: checking for `:destroy` events inside `:if`/`:unless` predicates

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [2.0.0] - 2019-04-01
```
Comment (tycooon) [inline CHANGELOG.md:4]: 2019?

File: lib/table_sync/version.rb [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module TableSync
-  VERSION = "1.13.1"
+  VERSION = "2.0.0"
```
Comment (VegetableProphet) [inline lib/table_sync/version.rb:4]: I don't think the changes are worth such huge version bump.

File: CHANGELOG.md (hunk 2 above)
Comment (EugeneYak) [reply CHANGELOG.md]: updated

**Pull request merged**